target/linux/mediatek/dts/mt7622-xiaomi-redmi-router-ax6s.dts

```diff
@@ -312,7 +312,7 @@
 			 */
 			partition@6c0000 {
```

Comment:
This unit address should also be updated.

Reply:
fixed